qutebrowser/mainwindow/tabbedbrowser.py

```diff
@@ -733,6 +737,18 @@ def on_scroll_pos_changed(self):
         self._update_window_title('scroll_pos')
         self.widget.update_tab_title(idx, 'scroll_pos')
 
+    @pyqtSlot()
```

Comment:
The `@pyqtSlot()` here doesn't actually do anything, because we use this with `functools.partial`.